client/src/containers/Home/ResendEmail/ResendEmail.js

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from "react";
+import { BrowserRouter as Router, Route, Link, NavLink } from "react-router-dom";
+import Select from "react-select";
+import { _resendEmail } from "../../../services/AuthService";
+import { connect } from "react-redux";
+import {bindActionCreators} from 'redux';
+import { _loadCryptocurrencies } from "../../../actions/loadCryptoActions";
+import { handleDropdownChange } from "../../../actions/signUpActions";
+import Footer from "../../../components/Layout/Footer";
+import Aside from '../Aside/Aside';
+
+class SignUp extends Component {
```

Comment:
Should this class be titled ResendEmail?
```suggestion
class ResendEmail extends Component {
```